hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconStorageContainerManagerFacade.java

```diff
@@ -366,7 +362,9 @@ public void start() {
           "Recon ScmDatanodeProtocol RPC server",
           getDatanodeProtocolServer().getDatanodeRpcAddress()));
     }
-    scheduler = Executors.newScheduledThreadPool(1);
+    scheduler = Executors.newScheduledThreadPool(1,
+        new ThreadFactoryBuilder().setNameFormat(threadNamePrefix)
```

Comment:
`threadNamePrefix` is `Recon-`.  We should include some other info in the thread name, e.g. `SyncSCM`.  If you have a better idea, please use that.  Also, let's add a numeric suffix to identify individual threads.

```suggestion
        new ThreadFactoryBuilder().setNameFormat(threadNamePrefix + "SyncSCM-%d")
```